Resources/Private/JavaScript/Terminal/src/helpers/fetchCommands.ts

```diff
@@ -14,7 +14,29 @@ const fetchCommands = async (endPoint: string): Promise<{ success: boolean; resu
                 'Content-Type': 'application/json',
             },
         }))
-        .then((response: Response) => response && response.json())
+        .then((response: Response) => {
+            if (!response.ok) {
+                return {
+                    success: false,
+                    result: [],
```

Comment:
The result has to be an object. I wonder why TS didn't complain.

```suggestion
                    result: {},
```